src/nvim/vim.h

```diff
@@ -23,7 +23,11 @@
 
 #include "nvim/os/os_defs.h"       // bring lots of system header files
 
-/// length of a buffer to store a number in ASCII (64 bits binary + NUL)
+// length of a buffer to store a number in ASCII (64 bits binary + NUL)
+// if this changes the max numberwidth tests in
+// test/functional/options/num_options_spec.lua and
+// src/nvim/testdir/test_options.vim
+// should be updated
```

Comment:
The tests will fail and that should make it obvious that they need to be updated, so we can avoid comments like this.

Reply:
Cool, I'll leave that out when I do the vim PR